hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleNetworkContext.java

```diff
@@ -218,36 +218,48 @@ public void deserialize(SerializableDataInputStream in, int version) throws IOEx
 		midnightRates = in.readSerializable(true, ratesSupplier);
 
 		if (version >= RELEASE_0130_VERSION) {
-			int numUsageSnapshots = in.readInt();
```

Comment:
If we're cleaning up this method, it might be a good time to remove always-true migration conditions (e.g. `version >= RELEASE_0130_VERSION` is always true now).

Reply:
Good idea! Removed the check of `version >= RELEASE_0130_VERSION`.

Reply:
Since we are now on release `0.16.0`, all the checks in this method will always be true; but not urgent to change in this PR.

Reply:
I think I can clean up `0.14.0` as well, but should we still keep `0.15.0` check? I mean, we may still need to restore from `0.15.0` state file, right?